src/api.cpp

```diff
@@ -2793,8 +2793,12 @@ cl_int CLVK_API_CALL clSetKernelArg(cl_kernel kern, cl_uint arg_index,
         return CL_INVALID_ARG_INDEX;
     }
 
+    // With opaque pointers, clspv is unable to infer the type of an unused
+    // kernel argument so just allow nullptr for its value. It will not have an
```

Comment:
```suggestion
    // kernel argument so allow nullptr for its value. It will not have an
```
I think the "just" is confusing as non-nullptr values AFAIU are accepted for unused arguments (they should be!).